ocs_ci/utility/utils.py

```diff
@@ -2313,14 +2348,55 @@ def ceph_health_check_base(namespace=None):
     """
     namespace = namespace or config.ENV_DATA["cluster_namespace"]
     health = run_ceph_health_cmd(namespace)
-
     if health.strip() == "HEALTH_OK":
         log.info("Ceph cluster health is HEALTH_OK.")
         return True
     else:
         raise CephHealthException(f"Ceph cluster health is not OK. Health: {health}")
 
 
+def ceph_health_check_base_multi_storagecluster():
+    """
+    Send ceph health to both internal and external cluster for multi-storagecluster purpose.
+
+    Raises:
+        CephHealthExceptionInternal: Internal cluster ceph health problem.
+        CephHealthExceptionExternal: External cluster ceph health problem.
+        CommandFailed: Incase command failed.
+        subprocess.TimeoutExpired: Incase timeout.
+        NoRunningCephToolBoxException: Incase toolbox doesn't exist - relevant only for internal cluster as
+                                       multi-storagecluster external uses ssh
+
+    Returns:
+        True: True if both ceph clusters are healthy.
```

Comment:
```suggestion
        bool: True if both ceph clusters are healthy.
```

Docstring should contains the returns type here, not specific value